modules/conversion-center/assets/scss/widgets/contact-buttons-var-9.scss

```diff
@@ -0,0 +1,276 @@
+.e-contact-buttons-var-9 {
+	--e-contact-buttons-size-small: 48px;
+	--e-contact-buttons-size-medium: 56px;
+	--e-contact-buttons-size-large: 64px;
+	--e-contact-buttons-svg-size-small: 24px;
+	--e-contact-buttons-svg-size-medium: 28px;
+	--e-contact-buttons-svg-size-large: 32px;
+	--e-contact-buttons-transition-duration: .5s;
+	--e-contact-buttons-transition: all var(--e-contact-buttons-transition-duration);
+	--e-contact-buttons-overlap-margin: -10px;
+
+	width: auto;
+
+	.e-contact-buttons {
+
+		&__chat-button-icon-container {
+			align-items: center;
+			background-color: var(--e-contact-buttons-button-bg);
+			border-radius: 50%;
+			display: flex;
+			justify-content: center;
+			position: relative;
+			transition: var(--e-contact-buttons-transition);
+			z-index: 1;
+
+			svg {
+				position: relative;
+				z-index: 2;
```

Comment:
@mserino it might be worth considering using the pre-defined layers
assets/dev/scss/helpers/variables.scss